contents/docs/product-analytics/group-analytics.mdx

```diff
@@ -24,10 +24,110 @@ import JSSetGroupProperties from "../\_snippets/setting-group-properties.mdx"
 
 <JSCreateGroups />
 
-### Setting and updating group properties
+## How to set group properties
 
 <JSSetGroupProperties />
 
+## How to capture group events
+
+Events are automatically linked to the groups that the user who submitted the event belongs to. Thus you do not add any additional code once you have linked a user to the group. This is the recommended way to capturing group events.
+
+To link a user to a group, call the same code as you would when [creating a group](#how-to-create-groups).
+
+### Advanced (server-side only): Capturing group events without a user
+
+If you want to capture group events but don't want to associate them with a specific user, there are two alternatives:
+
+#### Method one (recommended): Use the group ID as the event's distinct ID
+
+When capturing events, instead of using the user's ID as the distinct ID, you can use group ID as the distinct ID. For example:
+
+<MultiLanguage>
+
+```node
+posthog.capture({
+    event: 'group_event_name',
+    distinctId: 'group_id_in_your_db',
+})
+```
+
+```python
+posthog.capture('group_id_in_your_db', 
+    'group_event_name', 
+)
+```
+
+```go
+client.Enqueue(posthog.Capture{
+    DistinctId: "group_id_in_your_db",
+    Event: "group_event_name",
+})
+```
+
+```php
+PostHog::capture(array(
+    'distinctId' => 'group_id_in_your_db',
+    'event' => 'group_event_name'
+));
+```
+
+```bash
+curl -v -L --header "Content-Type: application/json" -d '{
+    "api_key": "<ph_project_api_key>",
+    "event": "group_event_name",
+    "distinct_id": "group_id_in_your_db",
+}' <ph_instance_address>/capture/
+```
+
+This has the advantage that insights such as "total users and "unique users" in [trends](/docs/product-analytics/trends) will work as expected for these groups events (note: although it will say "users", it will actually mean "groups")
```

Comment:
This assumes you are only using total users and unique users for groups and not individual users. If you do have individual users too, it gets messed up.

Reply:
This is per event though right? 
So depending on the event, "total users" would mean users or groups
e.g. for pageviews it would be users, but for `group_event` it would be group.

Perhaps I should clarify this in the text

Reply:
Fair point, there is also the `All events` series  to worry about.

Reply:
Since there's no link between users and groups, you shouldn't ever need to link these two together, even in insights.

If you want total count of events -> just do total count.
If you want unique count per group -> just do unique group.

user doesn't exist anyway, so no need to do total users.

If users do have meaning, then you should have used the distinct ID of the user in the first place? 🤔 . Am I missing something here?